libs/@local/hash-graph-client/python/graph_client/models.py

```diff
@@ -324,7 +317,7 @@ class UnarchivePropertyTypeRequest(BaseModel):
     type_to_unarchive: str = Field(..., alias="typeToUnarchive")
 
 
-class Schema4(BaseModel):
+class UpdateDataType(BaseModel):
```

Comment:
I liked the previous name (including `Request` in the name) more.

Reply:
still exists, this is simply the inner type

Reply:
Oh, got it. Sorry, the reordering confused me a lot 😅

Reply:
You are not the only one, don't worry^^